brainscore_language/metrics/linear_predictivity/test.py

```diff
@@ -4,7 +4,7 @@
 
 from brainio.assemblies import NeuroidAssembly
 from brainscore_language import load_metric
-from .metric import linear_regression
+from metric import linear_regression
```

Comment:
this is a convention we should discuss: 
* `from .metric` imports from the local package, which I think requires an `__init__.py`. For now, I left the inits out to discourage users from importing instead of using the plugin registry
* `from metric` assumes that the working directory is this directory so would not work if called from a parent directory

Reply:
I saw that you removed the `__init__.py`s but needed to choose between adding one here or switching out the relative import in order to get tests to pass. I chose the latter but am not attached to it - your call.

Reply:
let's do this for now, but bring it up in the next check-in